test/unittests/test_huffman.cpp

```diff
@@ -30,7 +30,7 @@ TEST_F(TestHuff, Ident2Bit) {
 
     encodeHuffman(tt);
 
-    ASSERT_TRUE(tt == ttExtend);
+    ASSERT_TRUE(TruthTable::equal(tt, ttExtend));
```

Comment:
This was the wrong way to add the equals call...
```suggestion
    ASSERT_TRUE(TruthTable::equal(tt, ttExtend, false));
```
would have been what is required to cover the missing line.